src/Factory/DelegatingNonceFactory.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace RebelCode\WordPress\Nonce\Factory;
+
+use Dhii\Factory\FactoryInterface;
+
+/**
+ * A nonce factory implementation that delegates to another generic factory.
+ *
+ * @since [*next-version*]
+ */
+class DelegatingNonceFactory extends AbstractNonceFactory implements NonceFactoryInterface
+{
+    /**
+     * The config array key to use to pass the nonce code to the delegate factory.
+     *
+     * @since [*next-version*]
+     */
+    const K_DELEGATE_CONFIG_ID = 'id';
```

Comment:
It's called differently: `K_CONFIG_ID`, the key of the ID in the config passed to a nonce factory. The `K_DELEGATE_*` constants are keys used when passing config to a generic factory.